.github/workflows/mulighetsrommet-arena-adapter-manager.yaml

```diff
@@ -42,14 +42,58 @@ jobs:
           channel_id: C026FHSRFEJ
           status: FAILED
           color: danger
+      - uses: actions/upload-artifact@v3
+        with:
+          name: dist
+          path: frontend/arena-adapter-manager/dist
```

Comment:
Hvorfor trenger vi denne?

Reply:
For å kunne ha byggsteget i et eget steg sånn at vi bare trenger å bygge en gang for prod og dev 🙌